schematic/manifest/generator.py

```diff
@@ -1458,56 +1419,59 @@ def export_sheet_to_excel(self, title: str = None, manifest_url : str = None, ou
-        if not output_format: 
-            # if the sheet_url parameter gets set to True, return a google sheet url 
-            if sheet_url: 
-                # populate google sheet with dataframe
-                manifest_sh = self.set_dataframe_by_url(empty_manifest_url, dataframe)
-                return manifest_sh.url
-                # else, return a pandas dataframe
-            else: 
-                return dataframe
-        else:
-            # if the output type gets set to "dataframe", return a data frame 
-            if output_format == "dataframe":
-                return dataframe
-            # if the output type gets set to "excel", return an excel spreadsheet
-            elif output_format == "excel":                 
-                # export the manifest url to excel
-                # note: the manifest url being used here is an empty manifest url that only contains column header
-
-                # export manifest that only contains column headers to Excel
-                output_file_path = self.export_sheet_to_excel(title = self.title, manifest_url = empty_manifest_url, output_location = output_path)
-                if not os.path.exists(output_file_path): 
-                    logger.error(f'Export to Excel spreadsheet fail. Please make sure that file path {output_file_path} is valid')
-
-                # populate an excel spreadsheet with the existing dataframe
-                self.populate_existing_excel_spreadsheet(output_file_path, dataframe)
 
-                return output_file_path
-            # for all other cases, return a google sheet url after populating the dataframe
-            else: 
-                manifest_sh = self.set_dataframe_by_url(empty_manifest_url, dataframe)
-                return manifest_sh.url
+        # if the output type gets set to "dataframe", return a data frame 
+        if output_format == "dataframe":
+            """
+            TODO: for backwards compatibility may need to return downloaded CSV only with no updates wrt to the schema. 
+            Probably use a different function and endpoint for this bc could just wrap a synapse client funtion instead of going through all the prior steps.
+            """
+            return dataframe
+        
+        # if the output type gets set to "excel", return an excel spreadsheet
+        elif output_format == "excel":              
+            # export manifest url that only contains column headers to Excel
+            output_file_path = self.export_sheet_to_excel(title = self.title,
+                                                          manifest_url = empty_manifest_url,
+                                                          output_location = output_path,
+                                                          )
+            
+            if not os.path.exists(output_file_path): 
+                logger.error(f'Export to Excel spreadsheet fail. Please make sure that file path {output_file_path} is valid')
```

Comment:
Maybe raise an error here is better than logging an error?

Reply:
This part was not actually added by me, but I can change to raise an error instead.

Reply:
I am actually going to move the path check upstream of this to the CLI. Check for it in an update.